doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -1071,14 +1071,6 @@ The :ref:`bluetooth_mesh_blob_cli` can be added to the mesh shell by enabling th
 	* ``Addr``: Unicast address of the Target node's BLOB Transfer Server model.
 
 
-``mesh models blob cli bounds [<Group>]``
```

Comment:
I guess it was renamed long time ago during the feature development:
https://github.com/zephyrproject-rtos/zephyr/blob/29208fe7c45c43997c5099718958f3535f33465b/subsys/bluetooth/mesh/shell/blob.c#L573

Reply:
Right you are, changed it to rename instead